main.go

```diff
@@ -24,6 +28,122 @@ type AvailableReleaseResponse struct {
 
 func main() {
 
+	if len(os.Args) == 1 {
+		InitHipo()
```

Comment:
InitHipo should run in all cases

We can handle zero param state with a usage message:

```
Usage:
hipo group:artifact:version
```